### PR TITLE
Remove 'Core' from ".NET Core SDK" and ".NET Core CLI"

### LINKED ISSUE #26726: ".NET Core SDK" --> ".NET SDK"?
Searching across this repository, the .NET docs have about 179 references to the ".NET Core SDK".  

In most cases I believe these should now be ".NET SDK" since that is the name used since .NET 5

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #26726

### CODE REVIEW

**Review by dsyme — APPROVED** [review]

File: docs/core/diagnostics/diagnostics-in-containers.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ WORKDIR /tools
 COPY --from=build /tools .
 ```
 
-Alternatively, the .NET Core SDK can be installed in a container when needed in order to install the CLI tools. Be aware that installing the .NET Core SDK will have the side-effect of reinstalling the .NET Core runtime. So be sure to install the version of the SDK that matches the runtime present in the container.
+Alternatively, the .NET SDK can be installed in a container when needed in order to install the CLI tools. Be aware that installing the .NET SDK will have the side-effect of reinstalling the .NET Core runtime. So be sure to install the version of the SDK that matches the runtime present in the container.
```
Comment (jkotas) [inline docs/core/diagnostics/diagnostics-in-containers.md:34]:
```suggestion
Alternatively, the .NET SDK can be installed in a container when needed in order to install the CLI tools. Be aware that installing the .NET SDK will have the side-effect of reinstalling the .NET runtime. So be sure to install the version of the SDK that matches the runtime present in the container.
```

File: docs/core/deploying/single-file.md [hunk 2]
```diff
@@ -222,11 +222,11 @@ For more information, see [Publish .NET Core apps with Visual Studio](deploy-wit
...
 ## Publish a single file app - Visual Studio for Mac
 
-Visual Studio for Mac doesn't provide options to publish your app as a single file. You'll need to publish manually by following the instructions from the [Publish a single file app - CLI](#publish-a-single-file-app---cli) section. For more information, see [Publish .NET Core apps with .NET Core CLI](deploy-with-cli.md).
+Visual Studio for Mac doesn't provide options to publish your app as a single file. You'll need to publish manually by following the instructions from the [Publish a single file app - CLI](#publish-a-single-file-app---cli) section. For more information, see [Publish .NET Core apps with .NET CLI](deploy-with-cli.md).
 
 ## See also
 
 - [.NET Core application deployment](index.md).
-- [Publish .NET Core apps with .NET Core CLI](deploy-with-cli.md).
+- [Publish .NET Core apps with .NET CLI](deploy-with-cli.md).
```
Comment (jkotas) [inline docs/core/deploying/single-file.md:230]:
```suggestion
- [Publish .NET apps with .NET CLI](deploy-with-cli.md).
```

File: docs/core/deploying/trimming/trim-self-contained.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ Note that trimming is only supported for self-contained apps.
 
 `dotnet publish -r win-x64 -p:PublishTrimmed=true`
 
-For more information, see [Publish .NET Core apps with .NET Core CLI](../deploy-with-cli.md).
+For more information, see [Publish .NET Core apps with .NET CLI](../deploy-with-cli.md).
```
Comment (jkotas) [inline docs/core/deploying/trimming/trim-self-contained.md:49]:
```suggestion
For more information, see [Publish .NET apps with .NET CLI](../deploy-with-cli.md).
```

File: docs/core/deploying/trimming/trim-self-contained.md [hunk 4]
```diff
@@ -76,11 +76,11 @@ For more information, see [Publish .NET Core apps with Visual Studio](../deploy-
 
 ### Publishing with Visual Studio for Mac
 
-Visual Studio for Mac doesn't provide options to publish your app. You'll need to publish manually by following the instructions from the [Publishing with the CLI](#publishing-with-the-cli) section. For more information, see [Publish .NET Core apps with .NET Core CLI](../deploy-with-cli.md).
+Visual Studio for Mac doesn't provide options to publish your app. You'll need to publish manually by following the instructions from the [Publishing with the CLI](#publishing-with-the-cli) section. For more information, see [Publish .NET Core apps with .NET CLI](../deploy-with-cli.md).
```
Comment (jkotas) [inline docs/core/deploying/trimming/trim-self-contained.md:79]:
```suggestion
Visual Studio for Mac doesn't provide options to publish your app. You'll need to publish manually by following the instructions from the [Publishing with the CLI](#publishing-with-the-cli) section. For more information, see [Publish .NET apps with .NET CLI](../deploy-with-cli.md).
```

File: docs/core/deploying/trimming/trim-self-contained.md [hunk 5]
```diff
@@ -76,11 +76,11 @@ For more information, see [Publish .NET Core apps with Visual Studio](../deploy-
...
 
 ## See also
 
 - [.NET Core application deployment](../index.md).
-- [Publish .NET Core apps with .NET Core CLI](../deploy-with-cli.md).
+- [Publish .NET Core apps with .NET CLI](../deploy-with-cli.md).
```
Comment (jkotas) [inline docs/core/deploying/trimming/trim-self-contained.md:84]:
```suggestion
- [Publish .NET apps with .NET CLI](../deploy-with-cli.md).
```

File: docs/core/deploying/single-file.md [hunk 6]
```diff
@@ -188,7 +188,7 @@ Publish a single file application using the [dotnet publish](../tools/dotnet-pub
 dotnet publish -r linux-x64 -p:PublishSingleFile=true --self-contained false
 ```
 
-For more information, see [Publish .NET Core apps with .NET Core CLI](deploy-with-cli.md).
+For more information, see [Publish .NET Core apps with .NET CLI](deploy-with-cli.md).
```
Comment (jkotas) [inline docs/core/deploying/single-file.md:191]:
```suggestion
For more information, see [Publish .NET apps with .NET CLI](deploy-with-cli.md).
```

File: docs/core/deploying/single-file.md [hunk 7]
```diff
@@ -222,11 +222,11 @@ For more information, see [Publish .NET Core apps with Visual Studio](deploy-wit
 
 ## Publish a single file app - Visual Studio for Mac
 
-Visual Studio for Mac doesn't provide options to publish your app as a single file. You'll need to publish manually by following the instructions from the [Publish a single file app - CLI](#publish-a-single-file-app---cli) section. For more information, see [Publish .NET Core apps with .NET Core CLI](deploy-with-cli.md).
+Visual Studio for Mac doesn't provide options to publish your app as a single file. You'll need to publish manually by following the instructions from the [Publish a single file app - CLI](#publish-a-single-file-app---cli) section. For more information, see [Publish .NET Core apps with .NET CLI](deploy-with-cli.md).
```
Comment (jkotas) [inline docs/core/deploying/single-file.md:225]:
```suggestion
Visual Studio for Mac doesn't provide options to publish your app as a single file. You'll need to publish manually by following the instructions from the [Publish a single file app - CLI](#publish-a-single-file-app---cli) section. For more information, see [Publish .NET apps with .NET CLI](deploy-with-cli.md).
```

**Review by jkotas — COMMENTED** [review]
Fix link text to match the linked doc title (the linked doc title does not have "Core")

**Review by jkotas — APPROVED** [review]

**Review by tommcdon — APPROVED** [review]

**Review by BillWagner — APPROVED** [review]
Thanks @tdykstra 

You can :shipit: when ready.

**Pull request merged**